ansible/devspace.yml

```diff
@@ -8,3 +8,15 @@
     - role: docker
     - role: versioncontrol-utils
     - role: devspace
+    - role: samba-client
+
+  tasks:
+
+    - name: Mount samba share
+      become: yes
+      mount:
+        fstype: cifs
+        name: /ome/data_repo
+        opts: "credentials=/etc/fstab.datarepo.creds,ro"
```

Comment:
I'm not sure what all the `.creds` file supplies, but addtional opts I've supplied via CLI `mount` are as follows:

`mount -v  -t cifs //squig.openmicroscopy.org.uk/ome-data-repo/ /opt/data_repo/ -o domain=domain-name,user=domain-user,uid=local-user-uid,gid=local-user-gid` cf https://trello.com/c/Xrc2wlMb/2705-trial-trout-or-octopus-read-only-jobs-using-samba